lib/utils.js

```diff
@@ -127,10 +127,48 @@ const config = values => {
   return result;
 };
 
+// Structures input from form
```

Comment:
Please add jsdoc code to the top of this. Should include a description `@param` and `@returns`